nbs/10_tutorial.pets.ipynb

```diff
@@ -114,27 +114,6 @@
     "We'll look at each `Transform` needed in turn. Here's how we can open an image file:"
```

Comment:
<p>why would you do that?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/fastai/pull/3380/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>